keps/sig-node/1539-hugepages/README.md

```diff
@@ -534,6 +542,12 @@ locality guarantees as a feature of QoS.  In particular, pods in the
 - E2E testing validating its usage.
 -- https://k8s-testgrid.appspot.com/sig-node-kubelet#node-kubelet-serial&include-filter-by-regex=Feature%3AHugePages
 
+## Graduation Criteria for HugePageStorageMediumSize
+
+- Reports of successful usage of the hugepage-<size> resources
+- E2E testing validating its usage
+-- https://k8s-testgrid.appspot.com/sig-node-kubelet#pr-kubelet-serial-gce-e2e-hugepages
```

Comment:
There don't appear to be e2es running here, were they moved?

Reply:
I'm able to see it. Can you try again?

Reply:
they moved to https://k8s-testgrid.appspot.com/sig-node-kubelet#kubelet-serial-gce-e2e-hugepages. I'll update the PR, thanks.